src/io/flutter/console/FlutterConsoleFilter.java

```diff
@@ -37,16 +73,27 @@ public Result applyFilter(final String line, final int entireLength) {
       return getFlutterDoctorResult(line, entireLength - line.length());
     }
 
+    boolean openAsExternalFile = false;
+
     String pathPart = line.trim();
 
     // Check for, e.g., "Launching lib/main.dart"
-    if (line.startsWith("Launching ")) {
-      final String[] parts = line.split(" ");
+    if (pathPart.startsWith("Launching ")) {
+      final String[] parts = pathPart.split(" ");
       if (parts.length > 1) {
         pathPart = parts[1];
       }
     }
 
+    // Check for, e.g., "open ios/Runner.xcworkspace"
```

Comment:
Do we want to lock this down a bit more? A more specific match, perhaps ensuring that what we're hyperlinking to ends in `.xcworkspace`?